src/applications/toe/helpers.jsx

```diff
@@ -245,3 +245,50 @@ export const applicantIsChildOfSponsor = formData => {
 
   return sponsor?.relationship === SPONSOR_RELATIONSHIP.CHILD;
 };
+
+export function transformTOEForm(_formConfig, form) {
+  const payload = {
+    claimant: {
+      claimantId: 0,
+      suffix: form?.data?.relativeFullName?.suffix,
```

Comment:
I'm assuming you copy/pasted all of the references to relativeFullName, relativeDateOfBirth, relativeAddress, etc? These should match up with data present in the form. In this case, userFullName, dataOfBirth, address, etc. Also, please use references to the formFields object in constants.js, as we do in form.js.

**NOTE**: I cleaned up the mappings in #21774. So, please rebase your branch from main.